test/example.cpp

```diff
@@ -5,22 +5,206 @@
 #include <string>
 #include <memory>
 #include "../3rdparty/gtest/googletest/include/gtest/gtest.h"
-#include "../include/myinteger.h"
-#include "../include/worker.h"
+#include "../include/mybiginteger.h"
+#include "../include/mathoperations.h"
+#include "../include/calculator.h"
 
-
-TEST(RPN, Is_RPN_Correct) {
+TEST(RPN_1, Is_RPN_1_Correct) {
   std::string exp = "-12+8-2*6 *( 9-3*6)*5-7+(2 - 1) +  99";
-  std::string exp2;
-  exp2.resize(100);
-  std::random_device rnd;
-  for (size_t i = 0; i < 100; ++i) {
-    char c = static_cast<char>(rnd() % 10 + '0');
-    exp2[i] = c;
-  }
-  Worker work(exp);
-  auto output = work.CalcExpression();
-  std::string out1 = output->GetValue();
-
-  EXPECT_EQ(out1, "629");
+  std::cout << "input expression#1: " << exp << "\n";
+  mycalc::Calculator work(exp);
+  auto output = work.CalcExpression();
+  std::cout << "answer#1: " << output->GetValue() << "\n";
```

Comment:
`endl` performs following actions:
1. `'\n'`
2. `fflush`
If you need fflush, than use it.